Test/MultiAssemblyAddin/MultiAssemblyAddin.csproj

```diff
@@ -52,5 +49,4 @@
       <Name>SecondAssembly</Name>
     </ProjectReference>
   </ItemGroup>
```

Comment:
To get this to compile I had to add some remove items. Otherwise I get errors about duplicate assembly attributes.

```
  <ItemGroup>
    <None Remove="OptionalModule\**" />
    <None Remove="SecondAssembly\**" />
    <Compile Remove="OptionalModule\**" />
    <Compile Remove="SecondAssembly\**" />
  </ItemGroup>
```